tools/RAiDER/cli/raider.py

```diff
@@ -76,7 +77,7 @@ def read_template_file(fname):
             params[key] = {}
 
     # Parse the user-provided arguments
-    template = DEFAULT_DICT
+    template = DEFAULT_DICT.copy()
```

Comment:
This is an extremely important piece that @jhkennedy help me added that ensures tests pass. The `DEFAULT_DICT` is mutable and if its changed during runtime (i.e. during tests) there are mind boggling consequences (might be worth changing default dict to a constructor (e.g. `frozen` dataclasss so it's constant, but for now this resolves this issue). There is a fixture in `test/test_checkArgs.py` that loads this. After adding this, tests that were failing, passed.